runtime/js/99_main.js

```diff
@@ -228,6 +229,7 @@ delete Object.prototype.__proto__;
     } else {
       prepareStackTrace = core.createPrepareStackTrace();
     }
+    // deno-lint-ignore prefer-primordials
     Error.prepareStackTrace = prepareStackTrace;
```

Comment:
In Node.js, `Error.prepareStackTrace` is skipped linting because of [the config](https://github.com/nodejs/node/blob/e4b1fb5e6422c1ff151234bb9de792d45dd88d87/lib/.eslintrc.yaml#L104-L105). But deno_lint doesn't support it right now, so just added an ignore directive here for the moment.